.github/workflows/main.yml

```diff
@@ -0,0 +1,12 @@
+on: check_suite
+name: Post Staging Link on pull_request
+jobs:
+  postStagingLink:
+    name: Post Staging Link
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Post Staging Link
+      uses: ./.github/action/result-poster
```

Comment:
AFAIK dockerfile support is gone, though I could be wrong?

Reply:
I believe docker is still supported with github actions.
The docs page has yml examples with docker containers, and I've used actions with docker in my personal projects.
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-a-docker-container-action

I apologize if that's different from the dockerfile you mentioned!